.github/workflows/remote-copy.yml

```diff
@@ -194,15 +190,14 @@ jobs:
         # and then one locally on the runner to copy the list of files from the remote.
         run: |
           ssh ${{ secrets.SSH_USER }}@${{ secrets.SSH_HOST }} -i ${{ steps.ssh.outputs.private-key-path }} /bin/bash <<'EOT'
-
           # Create intermediate directories since we can't use `rsync`s `--mkpath` on some deployment targets
-          if [ -d "${{ inputs.source }}" ]; then
-            mkdir -p "${{ inputs.target }}"
-          else  # we can assume we are transferring a file
-            mkdir -p $(dirname "${{ inputs.target }}")
-          fi
+          mkdir -p "${{ inputs.target }}"
```

Comment:
Do we care if the destination directory as a trailing slash? I don't *think* so, but would be good to test.

Reply:
It doesn't make a difference to `mkdir`, just verified